src/wrapper/bloom_callback.rs

```diff
@@ -187,17 +184,18 @@ fn external_vec_defrag(vec: Vec<u8>) -> Vec<u8> {
 /// Raw handler for the Bloom object's defrag callback.
 ///
 /// There are a few different structures we will be defragging we will explain them top down then afterwards state the order in which
-/// we will defrag. Starting from the top which is passed in as the variable named value. We have the BloomFilterType this BloomFilterType
-/// contains a vec of BloomFilters. These BloomFilters then each have a Bloom object. Finally each of these Bloom objects have a Vec.
+/// we will defrag. Starting from the top which is passed in as the variable named `value`, we have the BloomObject. This BloomObject
+/// contains a vec of BloomFilter structs. Each BloomFilter then each have a Bloom structure implemented in an external Rust crate).
```

Comment:
Missing a opening bracket in here
```suggestion
/// contains a vec of BloomFilter structs. Each BloomFilter then each have a Bloom structure (implemented in an external Rust crate).
```